### DISCUSSION TITLE
Live Chat doesn't update with new messages

### DISCUSSION THREAD

**Discussion by IzMichael** [chatwoot/chatwoot#1400] [Q&A]
Whenever someone sends a message, the other person needs to reload the page to receive it. Is this intentional, or if not, can someone help me fix it. I am hosting with my own Heroku deployment

**Comment by IzMichael** [comment]
Update: I've found #411 and #1322

**Comment by pranavrajs** [comment]
@IzMichael You have to turn on worker process. It is not turned on by default.

<img width="1230" alt="Screenshot 2020-11-12 at 9 42 29 AM" src="https://user-images.githubusercontent.com/2246121/98894711-6405e100-24cb-11eb-962a-7ec9b4e9b235.png">

**Comment by IzMichael** [comment]
Yep! works now, after turning on the worker process, maybe add this to the docs?
